### DISCUSSION TITLE
[WASM][.NET7] Strange error in browser console

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#10433] [Q&A] [answered]
I see this error in browser console
`uno-bootstrap.js:261 Uncaught (in promise) .NET runtime initialization failed (TypeError: Cannot read properties of null (reading 'querySelector'))`
.. when trying to start my app which I migrated from .NET6 to .NET7. 

This prevents the app from starting

Any idea what's going on?

**Comment by jeromelaban** [comment]
Which version of the uno bootstrapper are you running?

**Reply by DierkDroth** [reply]
@jeromelaban this is getting weird:
- if I'm using the exact index.html as per your recommendation [here ](https://github.com/unoplatform/uno/discussions/10415)then I'm getting the error as per first screenshot
- if I'm using a customized/cut down version of index.html as attached (since I don't want to have an 'splash' logic, I did the same thing with .NET6 which worked) then I'm getting error as per 2nd screenshot

My setup is as per 3rd screenshot

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/26179242/201846681-cd4d3d08-870a-4405-b22b-2d5a4f96246c.png">

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/26179242/201847899-ee4b52bc-1039-444a-a7ee-fd1a03b8eaa2.png">

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/26179242/201848073-32e9b0f3-5a4a-4837-ad4c-44e9541a40ce.png">

[index.zip](https://github.com/unoplatform/uno/files/10009974/index.zip)

**Reply by jeromelaban** [reply]
Which version of the uno bootstrapper are you running?

**Reply by DierkDroth** [reply]
@jeromelaban sorry I thought this was clear by my last screenshot: 7.0.7

Are are you looking for something different?

**Reply by jeromelaban** [reply]
I was on mobile, I missed the version. Then I do not know what causes this. If it works with no index.html override, try redoing your changes one by one.

**Reply by DierkDroth** [reply]
@jeromelaban above I tried to explain that I was testing both ways:
- standard index.html (no overwrite)
- customized index.html
Please see screenshots above for the different errors in both cases

Either way: I can't start my app.

**Reply by jeromelaban** [reply]
Thanks. If you can provide a repro, we'll be able to troubleshoot.

**Reply by DierkDroth** [reply]
@jeromelaban please understand this is my 'real world' app/project which I'm trying to migrate to .NET7. Obviously I simply can't upload that project as a repo.

On the other hand I would expect that I would not run into any such trouble starting with a fresh, blank sample project - apart from the fact that I haven't been able to create a fresh .NET7 project as per my other inquiry. Hence, this is not an option for a repo neither.

So what to do?

**Reply by jeromelaban** [reply]
We can only work on repro cases unfortunately, and we cannot guess what's breaking by using screenshots only.

**Comment by DierkDroth** [comment, marked as answer]
FYI for anyone who's running into that issue.

I was able to resolve by:
- applying launchSettings.json from a .NET7 blank project: the one I had with .NET6 did no longer work
- applying standard index.html + re-applying customizations as needed. The custom index.html I had with .NET6 no longer worked.
